.github/workflows/ci.yml

```diff
@@ -26,7 +26,7 @@ concurrency:
 
 jobs:
   tests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
I'd recommend using 22.04, that'll buy us more time before breaking

Reply:
Yup I resolved this in the meantime, I originally was going with a copy/paste from buildstream, which apparently recently changed to 22.04 :)